frontend/waterfall-trek-difficulty-map/index.html

```
@@ -0,0 +1,218 @@
+                        </div>
+                    </div>
+
+                    <div class="detail-panel">
+                        <div class="detail-card glass-card" id="detail-card" aria-live="polite">
+                            <div class="empty-state" id="empty-state">
+                                <span class="empty-icon">🥾</span>
+                                <p>Select a marker or a card below to view trek and accessibility details.</p>
+                            </div>
+
+                            <div class="waterfall-detail hidden" id="waterfall-detail">
+                                <img id="detail-image" class="detail-image" src="" alt="" loading="lazy" />
```

HTMLHint (1.9.2)

[error] The attribute [ src ] of the tag [ img ] must have a value.

(src-not-empty)